ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/target/TargetDefinition.java

```diff
@@ -627,9 +627,7 @@ private TargetBundle[] filterBundles(TargetBundle[] bundles, NameVersionDescript
 	 */
 	static List<TargetBundle> getMatchingBundles(TargetBundle[] collection, NameVersionDescriptor[] included, boolean handleMissingBundles) {
 		if (included == null) {
-			ArrayList<TargetBundle> result = new ArrayList<>();
-			result.addAll(Arrays.asList(collection));
-			return result;
+			return new ArrayList<>(Arrays.asList(collection));
```

Comment:
If the list should not be modified and you are sure the array doesn't contain null elements, you could also use `List.of()`.

Reply:
I'm not sure if null can happen or not, so my proposal would be to leave it as it.